WireDomain/Tests/WireDomainTests/LocalStores/TeamLocalStoreTests.swift

```diff
@@ -0,0 +1,331 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireDataModel
+import WireDataModelSupport
+@testable import WireDomain
+@testable import WireDomainSupport
+import XCTest
+import WireTestingPackage
+
+final class TeamLocalStoreTests: XCTestCase {
```

Comment:
basically all the local store tests is code that was moved out from repositories (with some adjustments sometimes)